lib/jelix-admin-modules/jauthdb_admin/controllers/default.classic.php

```diff
@@ -413,26 +448,44 @@ function delete(){
             jMessage::add(jLocale::get('crud.message.delete.invalid.pwd'), 'error');
             $rep->action = 'default:confirmdelete';
             $rep->params['j_user_login'] = $login;
+
             return $rep;
         }
 
-        if( $login !== null && jAuth::getUserSession()->login != $login){
-            if(jAuth::removeUser($login)) {
+        if ($login !== null && jAuth::getUserSession()->login != $login) {
+            if (jAuth::removeUser($login)) {
                 jMessage::add(jLocale::get('crud.message.delete.ok', $login), 'notice');
                 $rep->action = 'default:index';
-            }
-            else{
+            } else {
                 jMessage::add(jLocale::get('crud.message.delete.notok'), 'error');
                 $rep->action = 'default:view';
                 $rep->params['j_user_login'] = $login;
             }
-        }
-        else {
+        } else {
             jMessage::add(jLocale::get('crud.message.delete.notok'), 'error');
             $rep->action = 'default:index';
         }
+
         return $rep;
     }
 
-}
+    public function autocomplete()
+    {
+        $rep = $this->getResponse('text');
+
+        $dao = jDao::get($this->dao, $this->dbProfile);
+
+        $cond = jDao::createConditions();
+        $cond->addItemOrder('login', 'asc');
+        $list = $dao->findBy($cond);
+        $users = array();
+        foreach ($list as $prop) {
+            if (strstr($prop->login, $this->param('term')) || $this->param('term') === '') {
```

Comment:
if the term is empty or if its length is lower than 2 characters for exemple, you should return an empty JSON array, else it could returns hundred logins, this is not what we want.

Chek the content of term before using the dao